interface/clickmap/template/general_new.html

```diff
@@ -1,70 +1,64 @@
 {if !$reportMode}
     <html>
     <head>
-	{headerTemplate assets='jquery-ui|jquery-ui-lightness'}
+    {headerTemplate}
 {else}
-    {assetsTemplate assets='jquery-ui|jquery-ui-lightness'}
+    {headerTemplate}
```

Comment:
This line should be dropped (the assetsTemplate command only brings in the packages that are listed; and since none are listed anymore it can be removed).
The line in 4, however stays since headerTemplate brings in a bunch of defaults asset even when none are listed.